plugins/catalog-backend/api-report.md

```diff
@@ -52,38 +55,9 @@ import { UrlReader } from '@backstage/backend-common';
 import { Validators } from '@backstage/catalog-model';
 
 // @public (undocumented)
-export type AnalyzeLocationEntityField = {
-  field: string;
-  state:
-    | 'analysisSuggestedValue'
-    | 'analysisSuggestedNoValue'
-    | 'needsUserInput';
-  value: string | null;
-  description: string;
-};
-
-// @public
-export type AnalyzeLocationExistingEntity = {
```

Comment:
To not break the API we have need to keep exporting these.

However you should be able to deprecate them with
```
/**
 * @deprecated use the same type from `@backstage/plugin-x` instead
 * @public
 */
export type AnalyzeLocationExistingEntity = TypeFromCommonPackage;
```